NuKeeper/Git/LibGit2SharpDriver.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Threading.Tasks;
+using LibGit2Sharp;
+
+namespace NuKeeper.Git
+{
+    public class LibGit2SharpDriver : IGitDriver
+    {
+        private readonly string _repoStoragePath;
+        private readonly string _githubUser;
+        private readonly string _githubToken;
+
+        public LibGit2SharpDriver(string repoStoragePath, string githubUser, string githubToken)
+        {
+            _repoStoragePath = repoStoragePath;
+            _githubUser = githubUser;
+            _githubToken = githubToken;
+        }
+        public Task Clone(Uri pullEndpoint)
+        {
+            Repository.Clone(pullEndpoint.ToString(), _repoStoragePath);
+            return Task.CompletedTask;
```

Comment:
Is `LibGit2Sharp` likely to become async at any time soon?
Basically if not, and we discard the other git driver we should then make these methods sync.